### DISCUSSION TITLE
simple multiplayer with socket

### DISCUSSION THREAD

**Discussion by leer-ursina** [pokepetter/ursina#286] [General]
update file
[urs_multiplayer.zip](https://github.com/pokepetter/ursina/files/7318210/urs_multiplayer.zip)

chat is working now
moving, jumping are working 

hello there,
files is a multiplayer example for ursina 2D game

it use the old "socket" library and is local only

first unzip file (zipped with windows10)
launch server.py
click "start"
then
launch urs_client.py
launch urs_client_2.py

**Comment by leer-github** [comment]
hi there,
you have to launch client urs_client.py and urs_client2.py with a terminal for example
if you launch them from "idle" it does not work but server.py can be

server can send message to a player by writing in the white chat bar at the bottom and then clicking the player's name in the listbox

server.py
added=> player name deleted from listbox after being disconnected
used the new f-format

client files
fixed=> uncorrect player position by setting origin

[urs_multiplayer_1.zip](https://github.com/pokepetter/ursina/files/10804623/urs_multiplayer_1.zip)
